browser/infobars/sources.gni

```diff
@@ -64,9 +64,9 @@ if (!is_android) {
         "//brave/browser/infobars/brave_ipfs_infobar_delegate.h",
       ]
       brave_browser_infobar_deps += [
-        "//brave/browser/ui/views/infobars:infobars",
         "//brave/components/infobars/core:core",
         "//brave/components/vector_icons",
+        "//chrome/browser/ui",
```

Comment:
Seems we already have `"//chrome/browser/ui"` in the `brave_browser_infobar_deps` (added in the Ln 42)

Reply:
Should we have the same condition: `if (!is_android) {` for lines 12 and 57?

Reply:
> Seems we already have "//chrome/browser/ui" in the brave_browser_infobar_deps (added in the Ln 42)

Ah right, we can one of them

> Should we have the same condition: if (!is_android) { for lines 12 and 57?

Good point. It seems 57 is redundant to me as well.